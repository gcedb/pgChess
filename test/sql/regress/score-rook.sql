\set my_game 'k7/8/8/3p4/8/3R4/8/7K w - - 0 2'
\set pg2podg_iterations 1
\set pg2podg_depth_target 1

\ir _reset-games.sql
\ir _.sql
