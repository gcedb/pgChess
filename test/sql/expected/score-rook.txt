psql:test/sql/regress/_.sql:51: INFO:  1 turns to go (1 games)
ID: k7/8/8/3p4/8/3R4/8/7K w - - 0 2
psql:test/sql/regress/_.sql:51: INFO:  Score: -6.400
Choices:
	1.400 {} k7/8/8/3R4/8/8/8/7K b - - 0 2
	0.500 {} k7/8/8/3p4/8/1R6/8/7K b - - 1 2
	0.500 {} k7/8/8/3p4/8/3R4/6K1/8 b - - 1 2
	0.500 {} k7/8/8/3p4/8/R7/8/7K b - - 1 2
	0.300 {} k7/8/8/3p4/8/2R5/8/7K b - - 1 2
	0.300 {} k7/8/8/3p4/8/4R3/8/7K b - - 1 2
	0.300 {} k7/8/8/3p4/8/5R2/8/7K b - - 1 2
	0.300 {} k7/8/8/3p4/8/6R1/8/7K b - - 1 2
	0.200 {} k7/8/8/3p4/8/3R4/7K/8 b - - 1 2
	0.200 {} k7/8/8/3p4/8/3R4/8/6K1 b - - 1 2
	0.200 {} k7/8/8/3p4/8/7R/8/7K b - - 1 2
	0.100 {} k7/8/8/3p4/3R4/8/8/7K b - - 1 2
	0.000 {} k7/8/8/3p4/8/8/3R4/7K b - - 1 2
	-0.100 {} k7/8/8/3p4/8/8/8/3R3K b - - 1 2
psql:test/sql/regress/_.sql:51: INFO:  0 turns to go (15 games)
ID: k7/8/8/3R4/8/8/8/7K b - - 0 2
psql:test/sql/regress/_.sql:51: INFO:  game ended after 1 iterations
